routes/truckRoutes.js

```diff
@@ -1,3 +1,88 @@
 // connect to mongodb in this module as this is where you'll be making creat/read/delete calls to your database
 // use 'mongodb://localhost/foodTruckAPI' for your mongoose connection string
 // remember this is a Node module
+var express = require('express');
+var trucks = require('../models/truckModel');
+
+var mongoose = require('mongoose');
+var db = mongoose.connect('mongodb://localhost/bookAPI');
+var router = express.Router();
+
+
+
+router.route('/')
+	.get(function (request, response) {
+		Truck.find(function (error, trucks) {
+	        if (error) {   
+	            response.status(500).send(error);
+	        } else {
+	            response.json(trucks);
+	        }
+	    });
+	});

```

Comment:
Since you're chaining route handlers on your router object, you don't want a semi-colon here.